server/room-battle.js

```diff
@@ -520,7 +520,6 @@ class RoomBattle extends RoomGames.RoomGame {
 			this.addPlayer(options.p4, options.p4team || '', options.p4rating);
 		}
 		this.timer = new RoomBattleTimer(this);
-		if (Config.forcetimer) this.timer.start();
```

Comment:
I would set an `autostart` setting in the timer or something?

Reply:
WDYM? Like `this.timer.autostart = 3` so it starts the 3rd time `this.timer.nextRequest` is called, and then revert the changes below?

Reply:
I'll write this myself.